ui/pages/confirmations/types/confirm.ts

```diff
@@ -28,6 +28,18 @@ export type SignatureRequestType = {
     version?: string;
     siwe?: {
       isSIWEMessage: boolean;
+      parsedMessage: null | {
```

Comment:
As we're two levels deep with lots of properties, is it worth defining a separate type such as `SIWEData`?

Reply:
I did not needed it.

Reply:
I appreciate there is no usages, but just for readability, it can be nice to avoid complex nested types.